packages/antd/src/components/breadcrumb/index.tsx

```diff
@@ -38,9 +39,7 @@ export const Breadcrumb: React.FC<BreadcrumbProps> = ({
 
   const ActiveLink = routerType === "legacy" ? LegacyLink : Link;
 
-  if (breadcrumbs.length === 1) {
-    return null;
-  }
+  if (breadcrumbs.length < minItems) return null;
```

Comment:
- case 1: breadcrumbs.length = 0:  // both versions return null
- case 2: breadcrumbs.length = 1:  // old version returns null, new version shows
- case 3: breadcrumbs.length = 2:  // Both versions renders 

we have two options to fix case 2:
1. 
```suggestion
  if (breadcrumbs.length <= minItems) return null;
```
2. default should be set to 2 `minItems = 2`

I believe second option is better because it fits `minItems` name.

Reply:
I agree with point 1 but why should the default be set to 2? What is the rational behind this? If there is a configured route, the breadcrumb should show it be default unless instructed otherwise? 2 also seems arbitrary, why not3? :)

Reply:
Because previously, the breadcrumb would render only if there were at least 2 items, but it wouldn’t render if there was only 1 item. So, `minItems` should be 2. If we break this logic, it would be a breaking change, and current users who update Refine would see a breadcrumb even in cases with just 1 item.

I'm with you. If the route is configured, it should render, but we shouldn't make breaking changes, even if the old implementation is wrong.

Reply:
We should be good to merge then since I have the default for `minItems` to be `2`. Please let me know if there is another blocker.

Also for future major release might be a good idea to implement the breaking change of setting `minItems` to `1`.

Reply:
Everything is good, thanks but `should not render breadcrumb when the number of items is less than minItems` This test is not passing

Reply:
@alicanerdurmaz should be fixed now, kindly review.